src/app/shared/utils/dayjs.utils.ts

```diff
@@ -0,0 +1,48 @@
+import dayjs from 'dayjs';
+import {ManipulateTypeAlias as ManipulateType, UnitTypeAlias as UnitType} from '../types/dayjs-alias-type';
+import duration, {Duration} from 'dayjs/plugin/duration';
+import utc from 'dayjs/plugin/utc';
+import customParseFormat from 'dayjs/plugin/customParseFormat';
+
+dayjs.extend(duration);
+dayjs.extend(customParseFormat);
+dayjs.extend(utc);
+
+export class DayjsUtils {
+  public static getPartial(date: string, unit: UnitType): number {
+    return dayjs(date).get(unit);
+  }
+  public static getDateAsString(date: Date, format: string): string {
+    return dayjs(date).format(format);
+  }
+  public static getDate(date: string, format?: string): Date {
+    return format ? dayjs(date, format).toDate() : dayjs(date).toDate();
+  }
+  public static getUTCDateAsString(date: Date, format?: string): string {
+    return dayjs.utc(date).format(format);
+  }
+  public static getUnixDate(created: number): Date {
```

Comment:
It's not a unix date, though, it should be something like `getDateFromUnixTimestamp` or similar.